diff --git a/extras/Build/CMake/JUCEUtils.cmake b/extras/Build/CMake/JUCEUtils.cmake
index 9f0614e9b..8f7da8c2b 100644
--- a/extras/Build/CMake/JUCEUtils.cmake
+++ b/extras/Build/CMake/JUCEUtils.cmake
@@ -1210,13 +1210,16 @@ function(_juce_set_plugin_target_properties shared_code_target kind)
 
         get_target_property(vst3_auto_manifest ${shared_code_target} JUCE_VST3_AUTO_MANIFEST)
 
-        add_custom_command(TARGET ${target_name} POST_BUILD
-            COMMAND ${CMAKE_COMMAND} -E echo "removing moduleinfo.json"
-            COMMAND ${CMAKE_COMMAND} -E remove -f
-                "${output_path}/Contents/moduleinfo.json"
-                "${output_path}/Contents/Resources/moduleinfo.json")
+        #add_custom_command(TARGET ${target_name} POST_BUILD
+        #    COMMAND ${CMAKE_COMMAND} -E echo "removing moduleinfo.json"
+        #    COMMAND ${CMAKE_COMMAND} -E remove -f
+        #        "${output_path}/Contents/moduleinfo.json"
+        #        "${output_path}/Contents/Resources/moduleinfo.json")
+
+        message("skip module info")
 
         if(vst3_auto_manifest)
+            message("auto manifest")
             juce_enable_vst3_manifest_step(${shared_code_target})
         endif()
     elseif(kind STREQUAL "VST")
@@ -1327,6 +1330,7 @@ function(_juce_set_plugin_target_properties shared_code_target kind)
 
         _juce_set_copy_properties(${shared_code_target} ${target_name} "${output_path}" JUCE_LV2_COPY_DIR)
     endif()
+    message("endfunction")
 endfunction()
 
 # Place plugin wrapper targets alongside the shared code target in IDEs

